DirectEmployers/MyJobs
# [PD-3623] Preview Solr searches when making custom facets.

## What's this PR do?
Allows admins to preview Solr queries when creating custom facets.

## Screenshots (if appropriate)
![screen shot 2017-09-08 at 2 49 52 pm](https://user-images.githubusercontent.com/2728248/30226706-00506838-94a5-11e7-81ea-801cd8ae7a8b.png)
 
## Where should the reviewer start?
API is in seo/api/v2/
JavaScript is in the only .js file added.

## How should this be manually tested?
Import a few jobs.
Go to /admin/seo/customfacet/1/change/
Click the Advanced options (Show) toggle.
Click search and watch the results section magically populate. 
Now mess with a combination of search terms and business units. Changing those should impact the search results. Changing anything else should not. 

## Any background context you want to provide?
This was broken for a while, so the current workflow for DS is to create a facet, save, and then check it out on the site.
Because facet tagging will slow that process down quite a bit (since it requires a reimport), we opted to fix this as a happy medium between tagging on the fly and forcing DS to deal with how slow the process will be.
